### PR TITLE
h*: replace `plist` with `service` block

### PR DESCRIPTION
**Pull request by branchv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
See #79367

### CODE REVIEW

File: Formula/hbase.rb [hunk 1]
```diff
@@ -103,52 +103,18 @@ def post_install
...
+    keep_alive true
+    working_dir HOMEBREW_PREFIX
+    log_path var/"hbase/hbase.log"
+    error_log_path var/"hbase/hbase.err"
+    environment_variables HBASE_MASTER_OPTS: " -XX:PermSize=128m -XX:MaxPermSize=128m", HBASE_LOG_DIR: var/"hbase",
+                          HBASE_HOME: opt_libexec, HBASE_SECURITY_LOGGER: "INFO,RFAS", HBASE_PID_DIR: var/"run/hbase",
+                          HBASE_NICENESS: "0", HBASE_IDENT_STRING: "root",
+                          HBASE_REGIONSERVER_OPTS: " -XX:PermSize=128m -XX:MaxPermSize=128m",
+                          HBASE_OPTS: "-XX:+UseConcMarkSweepGC", HBASE_ROOT_LOGGER: "INFO,RFA",
+                          HBASE_LOG_PREFIX: "hbase-root-master", HBASE_LOGFILE: "hbase-root-master.log"
```
Comment (SMillerDev) [inline Formula/hbase.rb:117]: Can you put all arguments on one line?

Comment (branchvincent) [reply Formula/hbase.rb:117]: collapse this all down to a single line? that fails the audit: `Line is too long`

or do you mean each key/value pair on a separate line?

Comment (SMillerDev) [reply Formula/hbase.rb:117]: I meant each their own line.

**Review by SMillerDev — DISMISSED** [review]

**Comment by branchvincent** [comment]
gh actions issue: `Error: An error was encountered when uploading bottles. There were 1 items that failed to upload.`

rerunning...

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks for all the work @branchvincent !

**Pull request closed without merging**